paddle/fluid/operators/elementwise/mkldnn/elementwise_mkldnn_op.h

```diff
@@ -151,6 +157,13 @@ class EltwiseMKLDNNGradKernel : public ElemwiseGradKernel<T> {
     auto* dy = ctx.Output<Tensor>(framework::GradVarName("Y"));
     auto* dout = ctx.Input<Tensor>(framework::GradVarName("Out"));
 
+    // oneDNN's binary is optimized for broadcasting y into x, so in other case
```

Comment:
shouldn't we report to oneDNN request to optimize X broadcasting ? So we could could get rid of 
hacks at some point?

Reply:
I think that they are aware of that and they expect users to always broadcast Y into X, but we can create an issue for them. Anyway, for now we will need that workaround and we can delete it later if they will fix it